src/routes/cities.js

```diff
@@ -0,0 +1,59 @@
+import express from 'express';
+// for task 04
+// import fs from 'fs';
+
+import { CityModel } from '../models';
+import { createNotFindByIdError, createDBError, addModifiedDateTo } from '../utils';
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
+  // for task 04
+  // const citiesPath = `${__dirname}/../../assets/cities.json`;
+  // const cities = JSON.parse(fs.readFileSync(citiesPath, 'utf8'));
+  // const randomIndex = Math.floor(Math.random() * cities.length);
+
+  // return res.json(cities[randomIndex]);
+
+  CityModel.find({})
+    .then(cities => res.json(cities))
+    .catch(() => next(createDBError()));
+});
+
+router.post('/', (req, res, next) => {
+  const newCity = addModifiedDateTo(req.body);
+
+  new CityModel(newCity)
+    .save()
+    .then(city => res.json(city))
+    .catch(() => next(createDBError()));
```

Comment:
Good for start but not very informative. I'd separate validation error at least and show what is missing.